<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Крестики-нолики 5x5</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <!-- 25 кнопок 5x5 -->
      <item row="0" column="0"><widget class="QPushButton" name="pushButton_1"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="0" column="1"><widget class="QPushButton" name="pushButton_2"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="0" column="2"><widget class="QPushButton" name="pushButton_3"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="0" column="3"><widget class="QPushButton" name="pushButton_4"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="0" column="4"><widget class="QPushButton" name="pushButton_5"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="1" column="0"><widget class="QPushButton" name="pushButton_6"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="1" column="1"><widget class="QPushButton" name="pushButton_7"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="1" column="2"><widget class="QPushButton" name="pushButton_8"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="1" column="3"><widget class="QPushButton" name="pushButton_9"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="1" column="4"><widget class="QPushButton" name="pushButton_10"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="2" column="0"><widget class="QPushButton" name="pushButton_11"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="2" column="1"><widget class="QPushButton" name="pushButton_12"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="2" column="2"><widget class="QPushButton" name="pushButton_13"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="2" column="3"><widget class="QPushButton" name="pushButton_14"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="2" column="4"><widget class="QPushButton" name="pushButton_15"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="3" column="0"><widget class="QPushButton" name="pushButton_16"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="3" column="1"><widget class="QPushButton" name="pushButton_17"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="3" column="2"><widget class="QPushButton" name="pushButton_18"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="3" column="3"><widget class="QPushButton" name="pushButton_19"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="3" column="4"><widget class="QPushButton" name="pushButton_20"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="4" column="0"><widget class="QPushButton" name="pushButton_21"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="4" column="1"><widget class="QPushButton" name="pushButton_22"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="4" column="2"><widget class="QPushButton" name="pushButton_23"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="4" column="3"><widget class="QPushButton" name="pushButton_24"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
      <item row="4" column="4"><widget class="QPushButton" name="pushButton_25"><property name="flat"><bool>true</bool></property><property name="minimumSize"><size><width>100</width><height>100</height></size></property></widget></item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnSave">
        <property name="text"><string>Сохранить игру</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLoad">
        <property name="text"><string>Загрузить игру</string></property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnReset">
        <property name="text"><string>Сброс игры</string></property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text"><string>Результаты</string></property>
          <property name="alignment"><set>Qt::AlignCenter</set></property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item><widget class="QLabel" name="label"><property name="text"><string>Ходит</string></property></widget></item>
          <item><widget class="QLabel" name="label_2"><property name="text"><string/></property></widget></item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item><widget class="QLabel" name="labelCount"><property name="text"><string>Количество игр:</string></property></widget></item>
          <item><widget class="QLabel" name="labelCountRes"><property name="text"><string>0</string></property></widget></item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item><widget class="QLabel" name="labelPlayer1"><property name="text"><string>Игрок 1:</string></property></widget></item>
          <item><widget class="QLabel" name="labelPlayer1Res"><property name="text"><string>0</string></property></widget></item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item><widget class="QLabel" name="labelPlayer2"><property name="text"><string>Игрок 2:</string></property></widget></item>
          <item><widget class="QLabel" name="labelPlayer2Res"><property name="text"><string>0</string></property></widget></item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation"><enum>Qt::Vertical</enum></property>
        <property name="sizeHint" stdset="0"><size><width>20</width><height>40</height></size></property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>